### PR TITLE
OVA Webservers UI Enhancements

### PR DESCRIPTION
**Pull request by morris-jason**
Fixes: vmware/vic#5458
Fixes: vmware/vic-product#517
Fixes: vmware/vic-product#493

### CODE REVIEW

File: installer/Makefile [hunk 1]
```diff
@@ -21,7 +21,7 @@ SHA256SUM ?= sha256sum
 SED ?= sed
 RM ?= rm
 CP ?= cp
-OS := $(shell uname | tr '[:upper]' '[:lower:]')
+OS := $(shell uname | tr '[:upper:]' '[:lower:]')
```
Comment (andrewtchin) [inline installer/Makefile]: 👍

File: installer/fileserver/html/index.html [hunk 2]
```diff
@@ -67,7 +67,7 @@ <h1 id="getting-started">Getting Started</h1>
                                 <div class="card transparent">
                                     <div class="card-block">
                                         <div class="card-title">
-                                            <clr-icon shape="list" size="24"></clr-icon> Post-install & Deployment tasks
+                                            <clr-icon shape="list" size="24"></clr-icon> Post-Install & Deployment Tasks
```
Comment (andrewtchin) [inline installer/fileserver/html/index.html]: should this be `&amp;`?

File: installer/fileserver/server.go [hunk 3]
```diff
@@ -97,6 +106,29 @@ func Init(conf *config) {
...
+			log.Fatalf("impossible to fetch ovf environment, exiting")
+			os.Exit(1)
+		case lib.UnmarshalError:
+			fmt.Errorf("error: %s", err.Error())
+		}
+	}
+
+	if ip, err := ip.FirstIPv4(ip.Eth0Interface); err == nil {
+		conf.serverIP = ip
+		if strings.ToLower(ovf.Properties["management_portal.deploy"]) == "true" {
```
Comment (andrewtchin) [inline installer/fileserver/server.go]: Admiral is always deployed now so you can remove this if

File: installer/fileserver/html/index.html (hunk 2 above)
Comment (morris-jason) [inline installer/fileserver/html/index.html]: Not really sure what's going on here. I don't think it's valid HTML.

**Pull request merged**